Gemfile

```diff
@@ -73,6 +72,8 @@ gem 'capybara', :group => [:development, :test]
 gem 'paperclip', '~> 4.1'
 # as interactive debugger in error pages
 gem 'web-console', '~> 2.0', group: :development
+# Help with travis config testing
+gem 'travis', :group => [:development, :test]
```

Comment:
Too many dependencies just for a linter, we don't have to have this in the bundle I think.